guides/common/modules/proc_enabling-the-templatesync-plugin.adoc

```diff
@@ -9,3 +9,5 @@
 # {foreman-installer} --enable-foreman-plugin-templates
 ----
 . To verify that the plugin is installed correctly, ensure *Administer* > *Settings* includes the *TemplateSync* menu.
+. Optional: In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *TemplateSync* to configure the plugin.
```

Comment:
```suggestion
. Optional: In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Template Sync* to configure the plugin.
```

Reply:
Let's do this in a separate PR.

Reply:
https://github.com/theforeman/foreman-documentation/issues/3400